www/content/workstation/chef_shell.md

```diff
@@ -19,12 +19,12 @@ aliases = ["/chef_shell.html", "/chef_shell/"]
 The chef-shell executable is run as a command-line tool.
 
 Modes
-=====
+-----
```

Comment:
Why are we using .rst-style sections?

Reply:
Legacy and they're commonmark compliant. No reason other than that.